examples/twittermap/web/public/javascripts/map/controllers.js

```diff
@@ -47,6 +47,7 @@ angular.module('cloudberry.map', ['leaflet-directive', 'cloudberry.common','clou
     $scope.doNormalization = false;
     $scope.doSentiment = false;
     $scope.infoPromp = config.mapLegend;
+    $scope.addCityLayerRequest = false;
```

Comment:
Let's put a comment about this variable before declaring this. What does it do and why it is needed.

Reply:
And why don't we set it to true as default?